api/post.go

```diff
@@ -55,11 +56,15 @@ func createPost(c *Context, w http.ResponseWriter, r *http.Request) {
 
 		return
 	} else {
+		if result := <-Srv.Store.Channel().UpdateLastViewedAt(post.ChannelId, c.Session.UserId); result.Err != nil {
```

Comment:
I think you'll get unread action messages for yourself now?  stuff like "Corey joined this channel".  Not a big deal.